views-soy/src/test/groovy/io/micronaut/docs/SoySauceViewRendererSpec.groovy

```diff
@@ -95,7 +110,7 @@ class SoySauceViewRendererSpec extends Specification {
         rsp.body().contains("<h1>username: <span>sgammon</span></h1>")
     }
 
-    def "invoking /soy/missing should produce a 404 exception describing a missing view template"() {
+    def "invoking /soy/missing should produce a 500 exception describing a missing view template"() {
```

Comment:
soy now accurately returns ISE 500 for a missing explicitly-named template, rather than 404. this makes it consistent with the Freemarker modules and common sense (being that i am the author of the soy view engine, i am being salty at myself for this)